src/lectures/models/cmip.ipynb

```diff
@@ -0,0 +1,3361 @@
+{
```

Comment:
<p>Let's get some more introductory text about what this notebook will do. Also maybe something about what CMIP actually is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/earth-env-data-science/earth-env-data-science-book/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>My newest version has a nicer intro and generally some more guiding text</p><p> </p>